webapp/src/layouts/SidebarLayout/Header/Userbox/index.tsx

```diff
@@ -76,10 +76,10 @@ function HeaderUserbox() {
   return (
     <>
       <UserBoxButton color="secondary" ref={ref} onClick={handleOpen}>
-        <Avatar variant="rounded" alt={user.username} src={user.avatar} />
+        <FallbackAvatar fallback={`${user.firstname} ${user.lastname}`} variant="circular" alt={user.username} src={user.avatar} />
```

Comment:
create a const for this?

```ts
const userFullname = `${user.firstname} ${user.lastname}`
```


```suggestion
        <FallbackAvatar fallback={userFullname} variant="circular" alt={user.username} src={user.avatar} />
```